modules/olm-operator-lifecycle-manager.adoc

```diff
@@ -179,38 +181,341 @@ required to update a CSV to the latest version in a channel (stepping through
+|Contains the name of the OperatorGroup.
+
+|`olm.operatorGroupNamespace=<group_namespace>`
+|Contains the namespace of the OperatorGroup.
+
+|`olm.targetNamespaces=<target_namespaces>`
+|Contains a comma-delimited string listing the OperatorGroup's target namespace selection.
+|===
+
+NOTE: All annotations except `olm.targetNamespaces` are included with copied CSVs. Omitting the `olm.targetNamespaces` annotation on copied CSVs prevents the names of target namespaces from being leaked between tenants.
+
+[discrete]
+=== Provided APIs annotation
+
+Information about what `GroupVersionKinds` (GVKs) are provided by an OperatorGroup are shown in an `olm.providedAPIs` annotation. The annotation's value is a string consisting of `<kind>.<version>.<group>` delimited with commas. The GVKs of CRDs and APIServices provided by all active member CSVs of an OperatorGroup are included.
+
+Below is an example of an OperatorGroup with a single active member CSV providing the PackageManifest resource:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  annotations:
+    olm.providedAPIs: PackageManifest.v1alpha1.packages.apps.redhat.com
+  name: olm-operators
+  namespace: local
+  ...
+spec:
+  selector: {}
+  serviceAccount:
+    metadata:
+      creationTimestamp: null
+  targetNamespaces:
+  - local
+status:
+  lastUpdated: 2019-02-19T16:18:28Z
+  namespaces:
+  - local
+----
+
+[discrete]
+=== Role-based access control
+
+When an OperatorGroup is created, three ClusterRoles are generated. Each contains a single AggregationRule with a ClusterRoleSelector set to match a label, as shown below:
+
+[cols="1,1",options="header"]
+|===
+|ClusterRole |Label to match
+
+|`<operatorgroup_name>-admin`
+|`olm.opgroup.permissions/aggregate-to-admin: <operatorgroup_name>`
+
+|`<operatorgroup_name>-edit`
+|`olm.opgroup.permissions/aggregate-to-edit: <operatorgroup_name>`
+
+|`<operatorgroup_name>-view`
+|`olm.opgroup.permissions/aggregate-to-view: <operatorgroup_name>`
+|===
+
+When a CSV becomes an active member of an OperatorGroup and is not in a failed state with reason `InterOperatorGroupOwnerConflict`, the following RBAC resources are generated:
+
+* xref:resources-per-api-resource-crd[ClusterRoles for each API resource from a CRD]
+* xref:resources-per-api-resource-api[ClusterRoles for each API resource from an APIService]
+* xref:resources-additional-roles-rolebindings[Additional Roles and RoleBindings]
+
+[[resources-per-api-resource-crd]]
+.ClusterRoles generated for each API resource from a CRD
+[cols="1,1a",options="header"]
+|===
+|ClusterRole |Settings
+
+|`<kind>.<group>-<version>-admin`
+|Verbs on `<group>` `<kind>`:
+
+* `*`
+
+Aggregation labels:
+
+* `rbac.authorization.k8s.io/aggregate-to-admin: true`
+* `olm.opgroup.permissions/aggregate-to-admin: <operatorgroup_name>`
+
+|`<kind>.<group>-<version>-edit`
+|Verbs on `<group>` `<kind>`:
+
+* `create`
+* `update`
+* `patch`
+* `release`
```

Comment:
Not sure if I was looking at a good example ClusterRole, but this last one showed as "delete" instead of "release" when looking at the yaml. Should that be updated here (and below)?

Reply:
Yes, this should be `delete`. Nice catch!